### PR TITLE
solution

### PR DESCRIPTION
**Pull request by abcdesign1**
[DEMO LINK](https://vadym4che.github.io/react_article/)

### CODE REVIEW

**Review by Vlados3611 — APPROVED** [review]
Good work 👍

File: src/App.jsx [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 import 'bulma/css/bulma.css';
+// eslint-disable-next-line import/no-cycle
```
Comment (Andr1yK) [inline src/App.jsx:3]: Do you know why the linter gives an error?

**Review by Andr1yK — CHANGES_REQUESTED** [review]

File: src/App.jsx (hunk 1 above)
Comment (vadym4che) [reply src/App.jsx]: I've fixed it

File: src/App.jsx [hunk 2]
```diff
@@ -1,44 +1,25 @@
...
-          Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime
-          assumenda magni adipisci esse quidem ratione ipsa labore vero illo
-          optio.
-        </p>
+export function App() {
+  return (
+    <div className="section">
+      <div className="container">
+        <h1 className="title is-2">React Article</h1>
+        <Article
```
Comment (sergii-nosachenko) [inline src/App.jsx:17]:
```suggestion

        <Article
```

File: src/components/Article.jsx [hunk 3]
```diff
@@ -1 +1,40 @@
...
+    date,
+    text,
+  } = props;
+
+  return (
+    <div className="box">
+      <h2 className="title" data-cy="title">
+        {title}
+      </h2>
+      <p className="subtitle">
```
Comment (sergii-nosachenko) [inline src/components/Article.jsx:30]:
```suggestion

      <p className="subtitle">
```

File: src/components/Article.jsx [hunk 4]
```diff
@@ -1 +1,40 @@
...
+        <time dateTime={formatDate(date).dateISO} data-cy="date">
+          {formatDate(date).dateLocaleString}
+        </time>
+      </p>
+      <p data-cy="text">
```
Comment (sergii-nosachenko) [inline src/components/Article.jsx:35]:
```suggestion

      <p data-cy="text">
```

File: src/components/Article.jsx [hunk 5]
```diff
@@ -1 +1,40 @@
...
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  });
+
+  return {
+    dateISO,
+    dateLocaleString,
+  };
+};
```
Comment (sergii-nosachenko) [inline src/components/Article.jsx:16]: 👍

**Review by sergii-nosachenko — APPROVED** [review]

**Pull request closed without merging**